### PR TITLE
use an updated vector icons package to prevent componentdidmount warning

### PR DESCRIPTION
**Pull request by adamfitzgibbon**
✅ I have read the [contributing file](https://github.com/NMF-earth/nmf-app/blob/master/contributing.md)

## Summary

There wasn't an issue for this, but I noticed that there was often a "componentDidMount" warning message on app load. I found out we just had a slightly old version of the expo icons library so it just needed an update.

## Changelog

Installed the latest (that is supported by expo) version so we don't have that warning anymore.

## Demo

Just run the app and make sure you don't see the error involving "componentDidMount"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     ]
   },
   "dependencies": {
-    "@expo/vector-icons": "10.0.0",
+    "@expo/vector-icons": "^10.0.0",
```
Comment (ptbhatcoder) [inline package.json:29]: As per https://github.com/NMF-earth/nmf-app/blob/master/contributing.md#npm-packages, you might not use `^` or `~` or other semver chars. Could we freeze it to `10.0.2`

**Review by ptbhatcoder — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (adamfitzgibbon) [reply package.json:29]: Oh yeah good catch. I know it specifically says that this way we won’t have to use the lock file, but that’s the purpose of having a lock file. But no matter, I can get that changed tomorrow.

Comment (adamfitzgibbon) [reply package.json]: Fixed this. And it seems like 10.0.2 was still not quite new enough so setting it to 10.0.6 seemed to do it.

**Comment by PierreBresson** [comment]
@adamfitzgibbon merci :)

**Pull request merged**